custom_components/tahoma/coordinator.py

```diff
@@ -98,6 +98,16 @@ async def _async_update_data(self) -> Dict[str, Device]:
         for event in events:
             _LOGGER.debug(event)
 
+            if event.failure_type_code:
+                self.hass.bus.fire(
+                    "overkiz.event",
```

Comment:
@iMicknl what do you think about?

```suggestion
                    "overkiz.failure",
```

Or we try to be generic and just publish everything?

Reply:
Do you know how many events will be broadcasted if we publish everything? We don't want to fill the HA store with useless events / entries.

Reply:
If we publish everything, we will publish every events sent by Somfy. So most of the time there is nothing, except when you trigger an action. 
But for the moment, I don’t see a use case, except the failure one I gave as example. We can restrict the number of events today, and open later on demand.

Reply:
Makes sense, let's just do it for this use-case and document it. (for example how to notify an user / create a persistent notification).